hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java

```diff
@@ -82,6 +99,49 @@ private static long getTimeoutInSeconds(Class<?> clazz) {
         clazz.getName() + " does not have SmallTests/MediumTests/LargeTests in @Category");
   }
 
+  /**
+   * @param clazz Test class that is running.
+   * @return the number of parameters for this given test class. If the test is not parameterized or
+   *   if there is any issue determining the number of parameters, returns 1.
+   */
+  private static int getNumParameters(Class<?> clazz) {
+    RunWith[] runWiths = clazz.getAnnotationsByType(RunWith.class);
+    boolean isTestParameterized = runWiths != null && Arrays.stream(runWiths).anyMatch(
+      (r) -> r.value().equals(Parameterized.class));
+    if (isTestParameterized) {
+      for (Method method : clazz.getMethods()) {
```

Comment:
Style nit, rather than have an if with a big indented clause, I'd avoid the indent by testing for the opposite case and just doing an early return if succeeds -- especially when near the opening of a method. No biggie. Just a consideration for future.